application/src/main/data/upgrade/3.4.4/schema_update.sql

```diff
@@ -347,3 +347,13 @@ END
 $$;
 
 -- ALARM FUNCTIONS END
+
+CREATE TABLE IF NOT EXISTS entity_statistics (
+    entity_id uuid NOT NULL,
+    entity_type varchar(32) NOT NULL,
+    tenant_id uuid,
```

Comment:
tenant id is also not null, right?

Reply:
It is intended, for the future use with some entities not having the tenant id